main.go

```diff
@@ -85,7 +87,12 @@ var expectedExecutionMgrAddress common.Address
 var desiredExecutionMgrAddress = common.HexToAddress("00000000000000000000000000000000dead0000")
 var expectedStateMgrAddress common.Address
 var desiredStateMgrAddress = common.HexToAddress("00000000000000000000000000000000dead0001")
-var startingDeadAddress = common.HexToAddress("00000000000000000000000000000000dead0000")
+var startingDeadAddress = common.HexToAddress("00000000000000000000000000000000dead0002")
```

Comment:
We can make this `00000000000000000000000000000000dead0000` I realize. I forgot that just because the EM and SM are at dead0000 and dead0001, we can still start the dead addresses at dead0000